ReactAndroid/src/main/java/com/facebook/react/modules/blob/BlobModule.java

```diff
@@ -130,8 +131,21 @@ public boolean supports(String responseType) {
         }
 
         @Override
-        public WritableMap toResponseData(ResponseBody body) throws IOException {
-          byte[] data = body.bytes();
+        public WritableMap toResponseData(Response response) throws IOException {
+          byte[] data = {};
+          try {
+            data = response.body().bytes();
```

Comment:
What if we want to process a 204 no content? this would still fail when u call bytes on a null body

Reply:
Changed! Thank you for your suggestions 😄 !